web/exemple/blog/styles.css

```diff
@@ -0,0 +1,38 @@
+body {
```

Comment:
Nu trebuie sa modifici un fisier in acest director. Trebuie respectata structura prezentat la laborator.
La acest exercitiu trebuie sa copii exemplul acesta in directorul cu numele tau din `web/solutii/<nume_prenume>`

Pentru mai multe informatii acceseaza [pagina cursului](https://teaching.alexcoman.com/resurse/tutorial/git/2017/03/11/git-propunerea-unei-solutii/).